src/node_buffer.cc

```diff
@@ -730,12 +730,20 @@ void ReadDoubleBE(const FunctionCallbackInfo<Value>& args) {
 
 
 template <typename T, enum Endianness endianness>
-uint32_t WriteFloatGeneric(const FunctionCallbackInfo<Value>& args) {
+void WriteFloatGeneric(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+
+  bool noAssert = args[3]->BooleanValue();
+  if (!noAssert) {
```

Comment:
@matthewloring I think the convention in C++ is to use snake case for local variables and I am more interested in something like this

```c++
bool should_assert = !args[3]->BooleanValue();
...
if (should_assert) {
    ...
}
```